src/component.yaml

```diff
@@ -0,0 +1,37 @@
+%YAML 1.2
+---
+
+# package info
+id: csv-helper
+version: 2.10.0
+name: CSV Helper
```

Comment:
This file contains the Xamarin Component definition - similar to a nuspec for NuGet